gmso/core/dihedral.py

```diff
@@ -28,12 +29,24 @@ class Dihedral(Connection):
 
     __members_creator__: ClassVar[Callable] = Atom.model_validate
 
+    connectivity: ClassVar[Tuple[Tuple[int]]] = ((0, 1), (1, 2), (2, 3))
```

Comment:
Same, does this need a description and alias?

Reply:
See above.